### DISCUSSION TITLE
zk sync bridge problem

### DISCUSSION THREAD

**Discussion by Rajuttam15** [Cyfrin/foundry-full-course-cu#2675] [General]
unable to connect zk sync bridges to metamask

**Comment by EngrPips** [comment]
What error are you getting, @Rajuttam15?

**Reply by iam-qwerty** [reply]
I'm encountering the same issue.

![image](https://github.com/user-attachments/assets/daa5d28f-d578-4db2-ad29-22d1950bdab1)

**Reply by EngrPips** [reply]
Try updating your browser, then clear your browser cache and then retry.
